app/styles/ui/_dialog.scss

```diff
@@ -156,6 +157,7 @@ dialog {
         outline: 0;
       }
     }
+
```

Comment:
Can we exclude the white space changes from these scss files or is this some new prettier rule that's come into effect?

Reply:
Not sure, the same result after reformating in VS Code, kind of thinking these new added line breaks are correct, cause they are separate different selectors?
I can manually add them back if you think the old format is ok.